Nodejs/Product/TestAdapter/TestFrameworks/Angular/karmaConfig.js

```diff
@@ -19,6 +19,11 @@ module.exports = function (config) {
     // Replace all reporters
     config.reporters = ['vsKarmaReporter'];
 
+    if (!config.hostname && !config.listenAddress) {
+        // The default address is IPv4 but node 17+ uses IPv6. Update the value to 'localhost' (which works with both IPv4 and IPv6) if it was not set.
+        config.hostname = config.listenAddress = 'localhost';
```

Comment:
Karma documentation states that `hostname` is by default `localhost`. Do we need to set it up anyway?

Reply:
The test runner actually has a warning if listenAddress is set explicitly but hostname is not, so I just set them both explicitly to get rid of that warning.